WordPress/WordPressUITests/Tests/SignupTests.swift

```diff
@@ -26,4 +26,15 @@ class SignupTests: XCTestCase {
 
         XCTAssert(mySiteScreen.isLoaded())
     }
+
+    // Test Support Section Loads
+    // From Prologue > continue, tap "help" and make sure Support Screen loads
+    func testSupportScreenLoads() {
+        let supportScreen = PrologueScreen().selectContinue().selectHelp()
+
+        XCTAssert(supportScreen.isLoaded())
+
+        //close SupportScreen modal before teartown
+        SupportScreen().closeButton.tap()
```

Comment:
Have you considered using the screen object instance already available?

```suggestion
        supportScreen.closeButton.tap()
```